playbooks/portals-setup-following.yml

```diff
@@ -25,6 +25,9 @@
     - name: Include checking server OS compatibility
       include_tasks: tasks/check-server-os-compatibility.yml
 
+    - name: Load Custom Variable File
```

Comment:
We need to load vars also e.g. in `portals-setup-initial` and basically in every playbook and play where we set `lastpass_required: True`.

We should be able to do it simply using `vars_file`:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#defining-variables-in-included-files-and-roles

```
  vars:
    lastpass_required: True
  vars_file:
    - "{{ common_vars_file | default(omit) }}"
    - "{{ custom_vars_file | default(omit) }}"
```

Note that e.g. `portals-setup-following` playbook consists from 4 plays, we need to set it on all 4 plays.
And while `default(omit)` should work we should test it works as expected with `vars_file`.